src/coreclr/gc/gcconfig.h

```diff
@@ -65,7 +65,7 @@ class GCConfigStringHolder
 #define GC_CONFIGURATION_KEYS \
     BOOL_CONFIG  (ServerGC,               "gcServer",               NULL,                             false,             "Whether we should be using Server GC")                                                   \
     BOOL_CONFIG  (ConcurrentGC,           "gcConcurrent",           NULL,                             true,              "Whether we should be using Concurrent GC")                                               \
-    BOOL_CONFIG  (ConservativeGC,         "gcConservative",         NULL,                             false,             "Enables/Disables conservative GC")                                                       \
+    BOOL_CONFIG  (ConservativeGC,         "gcConservative",         NULL,                             true,              "Enables/Disables conservative GC")                                                       \
```

Comment:
I temporarily enabled conservative GC by default - for a CI run. That is not a part of the fix, just for test coverage. I will undo config changes.